src/main.cpp

```diff
@@ -54,6 +55,9 @@ static const char* MAIN_LOGGER_NAME = "Main";
 static const char* THREAD_LOGGER_NAME = "Thread";
 static const char* STATUS_LOGGER_NAME = "Status";
 
+// globally accesible metrics registry
+Snyder::MetricsRegistry* MetricsRegistry = new Snyder::MetricsRegistry();
```

Comment:
Can we abstain from globals if possible? Each such addition necessitates another addition in the [test runner](https://github.com/kyleds/driveshaft/blob/gearman_client_tests/tests/unit/tests.cpp#L34-L43) since the tests don't use main.cpp. Besides the fact that globals generally confound testing. 

It is preferable to instantiate it inside of main() and pass it wherever needed. Even better if MetricsRegistry has an interface defined that we can stub/mock in the testing environment. Alternatively, consider the Observer pattern wherever we want to generate metrics so as to decouple the application logic from the monitoring code.